docs/_snippets/webpack-final-to-vite-final.md

```diff
@@ -0,0 +1,83 @@
+```js filename=".storybook/main.js" renderer="common" language="js" tabTitle="With Webpack"
+export default {
+  // Replace your-framework with the framework you are using (e.g., react-webpack5, nextjs, angular)
+  framework: '@storybook/your-framework',
+  stories: ['../src/**/*.mdx', '../stories/**/*.stories.@(js|jsx|mjs|ts|tsx)'],
+  async webpackFinal(config) {
+    storybookBaseConfig.module?.rules?.push({
+      test: /\.(graphql|gql)$/,
+      include: [path.resolve('./lib/emails')],
+      exclude: /node_modules/,
+      loader: 'graphql-tag/loader',
+    });
+    storybookBaseConfig.module?.rules?.push({
+      test: /\.(graphql|gql)$/,
+      include: [path.resolve('./lib/schema')],
+      exclude: /node_modules/,
+      loader: 'raw-loader',
+    });
+
+    return config;
+  },
+};
+```
+
+```js filename=".storybook/main.js" renderer="common" language="js" tabTitle="With Vite"
+export default {
+  // Replace your-framework with the framework you are using (e.g., react-vite, vue3-vite)
+  framework: '@storybook/your-framework',
+  stories: ['../src/**/*.mdx', '../stories/**/*.stories.@(js|jsx|mjs|ts|tsx)'],
+  async viteFinal(config) {
+    return {
+      ...config,
+      plugins: [...(config.plugins ?? []), graphql()],
```

Comment:
@kylegach did you mean to add the import for the graphql plugin? or does it just represent the concept of a plugin?